scripts/requirements.txt

```diff
@@ -130,39 +150,35 @@ pygments==2.8.1
     #   ipython
 pykwalify==1.8.0
     # via west
-pyobjc-core==7.1 ; sys_platform == "darwin"
```

Comment:
Can you add these back in?

Managing this with multiple platforms is a bit tricky...

Reply:
Fixed. 
I missed that, would it be desirable run a multi-platform pip compile command like https://github.com/peterdemin/pip-compile-multi ?

Reply:
Maybe, does that solve the platform issues as well? It looks like it just allows a form of inheritance which I think doesn't quite solve the issues with platform specific packages.